clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/AuthRefreshTests.scala

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.clio.integrationtest.tests
+
+import akka.stream.scaladsl.Source
+import io.circe.Json
+import org.broadinstitute.clio.client.commands.ClioCommand
+import org.broadinstitute.clio.integrationtest.BaseIntegrationSpec
+import org.broadinstitute.clio.transfer.model.WgsUbamIndex
+
+import scala.concurrent.duration._
+import scala.util.Success
+
+trait AuthRefreshTests { self: BaseIntegrationSpec =>
+
+  behavior of "Clio Client"
+
+  it should "refresh the auth token after it expires" in {
+    /* Query the wgs-ubam schema every 3 seconds for 1.5 hours. */
```

Comment:
if the token expires after 1 hour, why are we testing 50% more time than the expiration time? Couldn't we try querying just once more after the token expires?

Reply:
I wanted to give it some wiggle room, but yeah this is probably overkill.